File: OLSource1/v1/VS_officedev/codesnippet/VisualBasic/walkthrough--importing-a-form-region-that-is-designed-in-outlook_3.vb
    Private Function FindTaskBySubjectName(ByVal subjectName As String) _
        As Outlook.TaskItem
        Dim Application As New Outlook.Application()
        Dim outlookNameSpace As Outlook.NameSpace = _
            Application.GetNamespace("MAPI")
        Dim tasksFolder As Outlook.MAPIFolder = _
            outlookNameSpace.GetDefaultFolder _
                (Microsoft.Office.Interop.Outlook.OlDefaultFolders.olFolderTasks)
        Dim taskItems As Outlook.Items = tasksFolder.Items
        Dim taskItem As Outlook.TaskItem
        For Each taskItem In taskItems
            If taskItem.Subject = subjectName Then
                Return taskItem
            End If
        Next taskItem
        Return Nothing
    End Function

File: OLSource1/v1/VS_officedev/codesnippet/CSharp/walkthrough--importing-a-form-region-that-is-designed-in-outlook_3.cs
        private Outlook.TaskItem FindTaskBySubjectName(string subjectName)
        {
            Outlook.Application Application = new Outlook.Application();
            Outlook.NameSpace outlookNameSpace = Application.GetNamespace("MAPI");
            Outlook.MAPIFolder tasksFolder =
                outlookNameSpace.GetDefaultFolder(
            Microsoft.Office.Interop.Outlook.
                OlDefaultFolders.olFolderTasks);
            Outlook.Items taskItems = tasksFolder.Items;
            foreach (Outlook.TaskItem taskItem in taskItems)
            {
                if (taskItem.Subject == subjectName)
                {
                    return taskItem;
                }
            }
            return null;
        }